spec/factories/subjects.rb

```diff
@@ -15,5 +15,10 @@
     factory :futher_education_subject do
       subject_name { 'Further Education' }
     end
+
+    factory :send_subject do
```

Comment:
We seem to be tending towards using traits in the other factories, does this make more sense as a factory?

Reply:
settled on making a factory for it as it related to actual record